.github/workflows/push_to_prod.yml

```
@@ -0,0 +1,48 @@
+name: Push to Production
+
+on:
+  workflow_call:
+    inputs:
+      image_name:
+        description: 'Docker image name'
+        required: true
+        type: string
+      service_name:
+        description: 'Service name'
+        required: true
+        type: string
+
+    secrets:
+      SSH_KEY:
+        description: 'SSH key used to access private repos during the build'
+        required: true
```

`SSH_KEY` isn't needed/used by the cloud run deployment SA is
```suggestion
      CLOUDRUN_DEPLOYER_SERVICEACCOUNT_KEY:
        description: 'Service account key used to trigger deployments to cloud run'
        required: true
```